data_structures/graphs/BFS_Travel.c

```diff
@@ -0,0 +1,87 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<limits.h>
+#include<string.h>
+
+
+//Structure for storing a graph
+struct Graph{
+	int vertexNum;
+	int** edges;
+};
+
+//Constructs a graph with V vertices and E edges
+void createGraph(struct Graph* G,int V){
+		G->vertexNum = V;
+		G->edges =(int**) malloc(V * sizeof(int*));
+		for(int i=0; i<V; i++){
+			G->edges[i] = (int*) malloc(V * sizeof(int));
+			for(int j=0; j<V; j++)
+				G->edges[i][j] = INT_MAX;
+			G->edges[i][i] = 0;
+		}		
+}
+
+//Adds the given edge to the graph 
+void addEdge(struct Graph* G, int src, int dst, int weight){
+	G->edges[src][dst] = weight;
+}
+
+//BFS Travel
+int row, col, visited[1000000], order[1000000];
+int head, tail;
+
+struct Data {
+	int now,v;
+	int enque(int cur, int lv) {
```

Comment:
We cannot have functions is struct in C. You may simulate functions using function pointers.